### PR TITLE
KOGITO-2710 Correct Jenkinsfile drools

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-2710

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: Jenkinsfile.drools [hunk 1]
```diff
@@ -15,66 +15,119 @@ pipeline {
...
+            }
+        }
+        stage('Build kogito-examples with persistence') {
+            steps {
                 dir("kogito-examples-persistence") {
                     script {
-                        // Don't run with tests so far, see: https://github.com/quarkusio/quarkus/issues/6885
-                        maven.runMavenWithSubmarineSettings("clean install ${MAVEN_EXTRA_ARGS} -Ppersistence", true)
+                        checkout(githubscm.resolveRepository("kogito-examples", "kiegroup", params.BRANCH_NAME, false))
+                        maven.runMavenWithSubmarineSettings('clean install -Ppersistence', false)
```
Comment (evacchi) [inline Jenkinsfile.drools:94]: is  `${MAVEN_EXTRA_ARGS}` not needed here?

**Comment by radtriste** [comment]
waiting for a full run of the pipeline before merging

File: Jenkinsfile.drools (hunk 1 above)
Comment (radtriste) [reply Jenkinsfile.drools:94]: it is, I will add it

**Comment by radtriste** [comment]
@cristianonicolai @evacchi Good to go
https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/kogito/job/custom-kogito-drools-snapshot/95/console

**Pull request merged**